pkg/csi-common/utils.go

```diff
@@ -111,3 +112,30 @@ func LogGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, h
 	}
 	return resp, err
 }
+
+func stripServiceAccountToken(req fmt.Stringer) string {
+	var parsed map[string]interface{}
+
+	err := json.Unmarshal([]byte(req.String()), &parsed)
+	if err != nil || parsed == nil {
+		return req.String()
+	}
+
+	volumeContext, ok := parsed["volume_context"].(map[string]interface{})
+	if !ok {
+		return req.String()
+	}
+
+	if _, ok := volumeContext["csi.storage.k8s.io/serviceAccount.tokens"]; !ok {
```

Comment:
let's make this func more generic, set `csi.storage.k8s.io/serviceAccount.tokens` as a parameter, thanks.

Reply:
rename as `stripSecrect(secretFieldNames []string)`

Reply:
ok, please take another look

Reply:
> rename as stripSecrect(secretFieldNames []string)

This will be conflict with the `StripSecrets` function in csi lib